Copilot/find instances of cite (#294)
symbiotic-engineering/MDOcean: pubs/renewable-energy-mdo/sections/other-appendices.tex

\subsection{Hessian Scaling Procedure}
\label{remdo:sec:appendix-scaling}
First the unscaled optimization is run for a small number (8) of iterations to obtain a realistic Hessian $H$.

> Then a scale vector $\textbf{s}$ with elements $s_i$ is calculated from the diagonal Hessian entries, as recommended in section 8.3 of reference text \citep{papalambros_principles_2017}:

Author: ```suggestion
Then a scale vector $\textbf{s}$ with elements $s_i$ is calculated from the diagonal Hessian entries, as \citet[section 8.3]{papalambros_principles_2017} recommend:
```

\begin{equation}
    A_{sens} \vec{x}_{sens}(p_i) = \vec{b}_{sens}(p_i)
\end{equation}

> The forms of the parameter-agnostic $A_{\text{sens}}$ matrix and the parameter-specific $b_{\text{sens}}$ vector are given in reference \citet{sobieszczanski-sobieski_sensitivity_1982} and not repeated here.

Author: ```suggestion
The forms of the parameter-agnostic $A_{\text{sens}}$ matrix and the parameter-specific $b_{\text{sens}}$ vector are given by \citet{sobieszczanski-sobieski_sensitivity_1982} and not repeated here.
```
